src/buildHTML.js

```diff
@@ -392,12 +439,17 @@ groupTypes.genfrac = function(group, options, prev) {
 
     var nstyle = style.fracNum();
     var dstyle = style.fracDen();
+    var newOptions;
 
-    var numer = buildGroup(group.value.numer, options.withStyle(nstyle));
-    var numerreset = makeSpan([style.reset(), nstyle.cls()], [numer]);
+    newOptions = options.withStyle(nstyle);
+    var numer = buildGroup(group.value.numer, newOptions);
+    var numerreset = makeSpan([style.reset(), nstyle.cls()],
+        [numer], newOptions);
```

Comment:
In that case it's probably best to either always pass `options`.  There are a few places where it's not being passed, `groupTypes.spacing`, `groupTypes.sqrt`, `groupTypes.array`, and `buildExpression`.  Can you add it to those call sites.